cosmos_rl/policy/config/__init__.py

```diff
@@ -856,24 +856,13 @@ def check_params_value(self):
         return self
 
 
-class RolloutParallelismConfig(ParallelismConfig):
+class RolloutParallelismConfig(BaseModel, extra="forbid"):
```

Comment:
seems still need based on Parallelsim? but we can add check for some dimension must be 1